scrapers_library/data_portals/ckan/README.md

```diff
@@ -0,0 +1,215 @@
+# CKAN Scraper
+
+## Introduction
+
+This scraper can be used to retrieve package information from [CKAN](https://ckan.org/), which hosts open data projects such as <https://data.gov/>. CKAN API documentation can be found at <https://docs.ckan.org/en/2.9/api/>.
+
+Running the scraper will output a list of packages to a CSV file using the search terms.
+
+## Definitions
+
+* `Package` - Also called a dataset, is a page containing relevant information about a dataset. For example, this page is a package: <https://catalog.data.gov/dataset/electric-vehicle-population-data>.
```

Comment:
These definitions are excellent and a welcome addition -- there's a lot of jargon with very specific meaning that could be easily confused, so this does a lot.